server/src-lib/Hasura/App.hs

```diff
@@ -193,8 +208,9 @@ runHGEServer
   -> InitCtx
   -> UTCTime
   -- ^ start time
+  -> ShutdownLatch
```

Comment:
This seems like it should probably go in the `InitCtx` with all the other odds and ends.